internal/app/document_store.go

```diff
@@ -79,50 +79,23 @@ func (s *documentStore) UpdateScanResults(ctx context.Context, lpaID, s3ObjectKe
 		"set VirusDetected = :virusDetected, Scanned = :scanned")
 }
 
-func (s *documentStore) BatchPut(ctx context.Context, documents []page.Document) error {
-	var converted []interface{}
-	for _, d := range documents {
-		converted = append(converted, d)
-	}
-
-	toWrite := len(converted)
-	written, err := s.dynamoClient.BatchPut(ctx, converted)
-
-	if err != nil {
-		return err
-	} else if written != toWrite {
-		return PartialBatchWriteError{Written: written, Expected: toWrite}
-	}
-
-	return nil
-}
-
 func (s *documentStore) Put(ctx context.Context, document page.Document) error {
 	return s.dynamoClient.Put(ctx, document)
 }
 
 func (s *documentStore) DeleteInfectedDocuments(ctx context.Context, documents page.Documents) error {
 	var dynamoKeys []dynamo.Key
-	var s3Keys []string
 
 	for _, d := range documents {
 		if d.VirusDetected {
-			dynamoKeys = append(dynamoKeys, dynamo.Key{
-				PK: d.PK,
-				SK: d.SK,
-			})
-			s3Keys = append(s3Keys, d.Key)
+			dynamoKeys = append(dynamoKeys, dynamo.Key{PK: d.PK, SK: d.SK})
 		}
 	}
 
 	if len(dynamoKeys) == 0 {
 		return nil
 	}
 
-	if err := s.s3Client.DeleteObjects(ctx, s3Keys); err != nil {
```

Comment:
not needed now https://github.com/ministryofjustice/opg-modernising-lpa/pull/822